CCHMapClusterController/CCHMapClusterOperation.h

```diff
@@ -41,6 +48,7 @@
 @property (nonatomic) id<CCHMapAnimator> animator;
 @property (nonatomic, weak) id<CCHMapClusterControllerDelegate> clusterControllerDelegate;
 @property (nonatomic, weak) CCHMapClusterController *clusterController;
+@property (nonatomic) CCHClusterMethod clusterMethod;
```

Comment:
Instead of having this type property I would rather create two classed called CCHMapClusterOperationDistanceBased and CCHMapClusterOperationGridBased